pandunia/3_buze_e_guye.md

```diff
@@ -1,52 +1,52 @@
-# tin buze e guye
+# tri mehe e gui
 
-<small>tarju da _Erjo_</small>
+<small>tarjum da _Erjo_</small>
 
 
-un ban, guye domu sa nice da pule, da transa suy.
-guye tena mal rupe. le tena kane – day ko plate, e nose – day ko warkobroxe.
+na tele pas, un gui be dom na nicha da un kiau, jo transe un rivo.
+la gui ye ten un dus rupa. ya ten oko jo dai ka tasa, e nos jo long ka bele.
 
-sa koy den, guye auda lil sonde sa supre da pule.
-«ki ren marca sa mi pule?» kritu guye.
-«o! mono me, lil buze» logu lil buze.
-le loga day xixo, e kontina:
-«me marca pa grasistan da me wil niama pa me si pangi».
+na un den, la gui ye audi un lili soni ze supra da kiau.
+«ke jan ye marche na mi su kiau?» be krita la gui.
```

Comment:
Better say: la gui ye krita, la lili behe ye loga, etc.

The old word order was motivated by the handiness of **-u**, but now writing "be krita la gui" sounds clumsy.